gslab_scons/release.py

```diff
@@ -61,11 +61,11 @@ def main(user_yaml = 'config_user.yaml',
 
     # Check provenance options
```

Comment:
I'd like users to be able to set verbose and (if possible) excluded_dirs as well. It seems silly to have these options that make a user's life easier, if the user can't access them in the standard use case.

Reply:
`excluded_dirs` can't be set from command line since it's a list. `verbose` is fine.

Reply:
It could be passed as a string, which we convert into a list, but yeah that might be a little much. I'm also not positive of the use case for that argument.